app/controllers/report_controller.rb

```diff
@@ -274,6 +274,19 @@ def title
 
   private
 
+  # Get string with unavailable fields for displaying note about them while adding/editing report
+  def unavailable_fields_for_model
+    case @edit[:new][:model]
+    when 'ChargebackVm'
+      'Allocated/Used CPU Cores'
+    when 'ChargebackContainerImage'
+      'Allocated CPU Count, Used CPU, Used Disk I/O and Used Network I/O'
+    when 'ChargebackContainerProject'
+      'Allocated CPU Count/Cores, Used CPU, Used Disk I/O and Allocated Memory'
+    end
```

Comment:
Translations?

Reply:
I wouldn't use `ui_lookup` here. Or did you mean anything else?

Reply:
`_`